### PR TITLE
Update wovn_helper.php for coding standards

### PR DESCRIPTION
**Pull request by a-bougouin**
### Purpose/goal of PR
Update wovn_helper.php for coding standards

### CODE REVIEW

File: .phpcs.xml [hunk 1]
```diff
@@ -9,7 +9,6 @@
   <!-- FIXME: original indentation is 4, but it was changed to 2 -->
   <exclude-pattern>./src/wovnio/modified_vendor</exclude-pattern>
   <!-- TODO: fix excluded files below -->
```
Comment (suplo) [inline .phpcs.xml]: this line should be removed as well?

Comment (a-bougouin) [reply .phpcs.xml]: I'll remove later, in the indentation PR.

Comment (suplo) [reply .phpcs.xml]: ok

Comment (a-bougouin) [reply .phpcs.xml]: thanks

Comment (a-bougouin) [reply .phpcs.xml]: did by resolving conflicts actually ;)

**Pull request merged**